public/arrowRightWhite.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 31 12"><path opacity=".8" d="M30.53 6.53a.75.75 0 000-1.06L25.757.697a.75.75 0 00-1.06 1.06L28.939 6l-4.242 4.243a.75.75 0 001.06 1.06L30.53 6.53zM0 6.75h30v-1.5H0v1.5z" fill="#fff"/></svg>
```

Comment:
'fill' is a CSS property you can set. If the fill is the only change from the other arrows, you can remove this and set a class that updates/changes the fill.